android_mic_streaming/app/src/main/java/org/mozilla/deepspeechdemo/MainActivity.kt

```diff
@@ -0,0 +1,137 @@
+package org.mozilla.deepspeechdemo
+
+import android.Manifest
+import android.content.pm.PackageManager
+import android.media.AudioFormat
+import android.media.AudioRecord
+import android.media.MediaRecorder
+import android.os.Build
+import android.os.Bundle
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.app.ActivityCompat
+import kotlinx.android.synthetic.main.activity_main.*
+import org.mozilla.deepspeech.libdeepspeech.DeepSpeechModel
+import org.mozilla.deepspeech.libdeepspeech.DeepSpeechStreamingState
+
+
+class MainActivity : AppCompatActivity() {
+    private var model: DeepSpeechModel? = null
+    private var streamContext: DeepSpeechStreamingState? = null
+
+    private val BEAM_WIDTH = 50
```

Comment:
That feels very small

Reply:
Was is appropriate? You had 500 in the french example.
I guess I'll look into the other examples using english language.

Reply:
I think you can stick to the default value of 500 we have everywhere

Reply:
Okay I found the value 500 in [python mic streaming this example](https://github.com/erksch/DeepSpeech-examples/blob/android-mic-streaming/mic_vad_streaming/mic_vad_streaming.py). I'll use that.